sources/sdk/k8s-operator/tests/kube-interface/index.spec.js

```diff
@@ -33,10 +33,12 @@ describe('KubeInterface', () => {
     )
   })
 
-  require('./create')()
-  require('./list')()
-  require('./get')()
-  require('./watch')()
-  require('./patch')()
-  require('./delete')()
+  describe('create', require('./create'))
+  describe('list', require('./list'))
+  describe('get', require('./get'))
+  describe('watch', require('./watch'))
+  describe('patch', require('./patch'))
+  describe('delete', require('./delete'))
+  describe('logs', require('./logs'))
+  describe.skip('exec', require('./exec'))
```

Comment:
Skipped because the dependency `kubernetes-client` is buggy (godaddy/kubernetes-client#570)